appliance_specific/index.md

```diff
@@ -0,0 +1,40 @@
+---
+---
+
+# Backup
+Creating a backup of the appliance will be done using SSH or the console.
+
+1. Launch `appliance_console`
+  a. Select `12) Stop EVM Server Processes` to stop the server processes
+  b. Select `2) Create Database Backup` and choosea backup location (/tmp/evm_db.backup is the default)
+  c. Select `17) Quit` to return to the shell
```

Comment:
Minor, but I suggest remove the 12), 2), etc, because the numbers could change if we insert new stuff (think for Kafka).  I think the text is sufficient.